src/Maker/MakeEntity.php

```diff
@@ -97,6 +97,7 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
             ->addOption('broadcast', 'b', InputOption::VALUE_NONE, 'Add the ability to broadcast entity updates using Symfony UX Turbo?')
             ->addOption('regenerate', null, InputOption::VALUE_NONE, 'Instead of adding new fields, simply generate the methods (e.g. getter/setter) for existing fields')
             ->addOption('overwrite', null, InputOption::VALUE_NONE, 'Overwrite any existing getter/setter methods')
+            ->addOption('table-name', 't', InputOption::VALUE_NONE, 'Overwrite default table name')
```

Comment:
Overwrite seems more something to change existing things no ? 

I'd say this will be also (mainly?) used during first Entitiy creation ? 

Maybe something more like "Set custom table name" ? 

Feel free to ignore if you disagree, just a comment passing by ;)

Reply:
To be honest I hadn't thought that far ahead. But I agree with you for this change, sounds better